ci/nightly/pipeline.template.yml

```diff
@@ -456,6 +456,10 @@ steps:
       - id: aws-real
         label: AWS (Real)
         timeout_in_minutes: 30
+        retry:
+          automatic:
+            - exit_status: 1
```

Comment:
Why explicitly exit_status 1 instead of any other than 0, which is the default I think?

Reply:
Are you suggesting to drop this line?

Reply:
Actually `exit_status: "*"`: https://buildkite.com/docs/pipelines/command-step#retry-attributes-automatic-retry-attributes